modules/oxide/src/less/theme/components/dialog/dialog.less

```diff
@@ -40,11 +40,13 @@
 @dialog-body-text-transform: none;
 
 @dialog-body-link-color: @color-tint;
-@dialog-body-link-text-decoration: none;
-@dialog-body-link-hover-color: darken(@color-tint, 10%);
-@dialog-body-link-hover-text-decoration: none;
-@dialog-body-link-active-color: darken(@color-tint, 10%);
-@dialog-body-link-active-text-decoration: none;
+@dialog-body-link-text-decoration: underline;
```

Comment:
That’s totally up to you, but imo having links permanently underlined would make plugins list usecase mentioned in the Jira look rather busy. Though I agree that having links underlined before hover (and possibly removing the underline on hover) is a better for the a11y perspective, so I guess the final decision would depend on which situation is more likely yo be encountered: lists of links or a single link inside a paragraph as in the example in the PR.

Reply:
I think we end on the side of being more accessible with the default skin. It makes it easier to identify them as links even if you don't see color at all.